website-1.x/pages/en/index.js

```diff
@@ -62,12 +62,14 @@ class Index extends React.Component {
         <HomeSplash siteConfig={siteConfig} language={language} />
         <div className="announcement">
           <div className="announcement-inner">
-            We're working on{' '}
-            <a href="https://github.com/facebook/docusaurus/issues/789">
-              Docusaurus 2
+            If you don't need advanced features such as versioning or
```

Comment:
> versioning

This is no longer relevant, is it?

Reply:
I don't think V2 versioning is battle tested yet